### DISCUSSION TITLE
Why is a 7.0.4 released after 7.1.1

### DISCUSSION THREAD

**Discussion by Mister-Hope** [PowerShell/PowerShell#14639] [Q&A] [answered]
I am a bit confused?

**Comment by qContinuum** [comment]
Same here... 🤦‍♂️

**Comment by pepoluan** [comment]
Probably for those who had to use 7.0 because of subtle incompatibilities with 7.1.

**Comment by ScrambledBrain** [comment, marked as answer]
Generally there are multiple versions in the development as usual:

- PowerShell 7[.0.x] is the current LTS release
- PowerShell 7.1[.x] is the current stable release
- PowerShell 7.2[.x] is the current preview release

So, you will see different release versions, because they are all maintained.

But the releases could have better information about the release type by adding a name extension or a tag for LTS/stable/preview.
"Latest" GitHub release is maybe not the matching release for your needs ;D

**Reply by ronascentes** [reply]
I agree. The releases could have a better information. Even I'm a advanced PowerShell user, I get confused some times about the mix of versions. For enterprise, we need to have a central place to get the LTS version quickly like ASP.NET download page.
![image](https://user-images.githubusercontent.com/8158428/108396111-41cfdd80-71f5-11eb-9251-213780f56a36.png)

**Comment by Dustyn69** [comment]
Yeah the version numbers are very confusing now.
So 7.1.1 is still the latest version?
I'm assuming you would not want to install 7.0.4 ontop of 7.1.1?

**Reply by pepoluan** [reply]
Basically, you try.

Install 7.1.1. If your scripts work, then good.

If not, go down a rung and install 7.0.4. That way, you'll benefit from security fixes although you'll be missing out new features.

**Comment by SteveL-MSFT** [comment]
Releases are typically aligned with .NET releases.  These happen concurrently (so .NET Core 3.1.x and .NET 5.x come out around the same time), so the PowerShell Team has to work on 7.0.x and 7.1.x releases concurrently (we would have had a 7.2 preview but explicitly decided not to this month).  In this case, we hit some issues with the 7.0.x release which required more time since we've made improvements to 7.1.x release pipeline that wasn't backported to 7.0.x.  The timing of the releases is completely independent of LTS vs non-LTS, etc...  Our goal is to have simultaneous releases with .NET, but we aren't there yet.

**Comment by otterdas** [comment]
This explanation helps very much, but it took a lot of effort to find.  You may want to be thoughtful of public links.  [Joey Aiello's Announcement of PowerShell 7.1 on devblogs](https://devblogs.microsoft.com/powershell/announcing-powershell-7-1/) includes a statement with a link that "Our latest releases can always be found within the GitHub Releases for PowerShell"; however, that link now leads to the 7.0.4 build.  For they lay moron like me, it's all very confusing.   I'm still trying to find  the 7.1.1 link.

**Reply by DiZzyDMon** [reply]
On the main PowerShell GitHub page https://github.com/PowerShell/PowerShell/releases, click on the "Compare" button, find 7.1.1 in the list.
